app/models/ability.rb

```diff
@@ -57,6 +57,8 @@ def initialize(user)
       user.admin? || (preference.team.students.include? user)
     end
 
+    can :crud, Conference if user.admin? || user.current_student?
```

Comment:
According to the comments on this PR: shouldn't this only be allowed for students from accepted teams?

Reply:
`user.current_student?` returns students from accepted teams.

```
  def current_student?
    roles.joins(:team).
      where("teams.season_id" => Season.current.id, "teams.kind" => %w(sponsored voluntary)).
      student.any?
  end
```

Reply:
Oh ok, looks fine, thanx @juuh42dias 🙏